src/cmd/build.js

```diff
@@ -52,7 +52,8 @@ function defaultPackageCreator(
 
 export default function build(
     {sourceDir, artifactsDir, asNeeded=false}: Object,
-    {manifestData, fileFilter=new FileFilter(),
+    {manifestData, fileFilter=new FileFilter({
+      filePathsToIgnore: [path.resolve(artifactsDir)]}),
```

Comment:
This is getting hard to read. I suggest moving it the execution body of the function. In other words, something more like:
````javascript
export default function build({manifestData, fileFilter}) {
  if (!fileFilter) {
    fileFilter = new FileFilter({
      filePathsToIgnore: [path.resolve(artifactsDir)],
    });
  }
}
````